### ISSUE TITLE
Legends button doesn't work

### ISSUE THREAD

**Issue by yeahphil** [AmericanWhitewater/aw-ios#232]
There's a legends view that's supposed to be displayed when you tap the info button at upper left on the runs list controller. Rough screenshot attached. This doesn't work right now because it's behind the table view so is hidden by it.

Is this supposed to appear?

<img width="419" alt="image" src="https://user-images.githubusercontent.com/66321/127272732-e05eea8b-01c4-40a7-b457-65c7696a4e18.png">

**Comment by Gregliest** [comment]
Yeah that's a bug.

**Comment by AWKevin** [comment]
This does not appear to be fixed. I don't see the Legend with I click the 'I" in the upper left corner of the runs list. As far as I can tell there is no legend anywhere describing what the colors mean.

**Comment by AWKevin** [comment]
Now I can't see the little "I" anywhere. Is it now behind the "Last Updated Bar?" I searched all pages I could think of to find a legend to describe the colors and could not find one.

**Comment by AWKevin** [comment]
Yes it is very important IMO. We have color coded flow readings on the App but nothing to tell visitors what that means. Just as importantly when you boil down the AW value-add to what else is out there it is our info on what levels are good or not good.  I think anywhere there is color-coded flow info we should provide access to a legend.

**Comment by Gregliest** [comment]
@yeahphil If you've got time, would you mind tackling this one?
